sdk/metric/view/view.go

```diff
@@ -17,22 +17,195 @@
 
 package view // import "go.opentelemetry.io/otel/sdk/metric/view"
 
-// Config contains configuration options for a view.
-type Config struct {
-	// TODO (#2837): implement.
+import (
+	"fmt"
+	"regexp"
+	"strings"
+
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/sdk/instrumentation"
+)
+
+// View provides users with the flexibility to customize the metrics that are
+// output by the SDK. A View can be used to:
+//
+// * Ignore Instruments.
+// * Change the name of an Instrument.
+// * Change the aggregation of an Instrument.
+// * Customize which attribute(s) are to be reported by the Instrument.
+//
+// An empty config will match all instruments, and do no modifications.
+type View struct {
+	instrumentName *regexp.Regexp
+	hasWildcard    bool
+	scope          instrumentation.Library
+
+	filterKeys  map[attribute.Key]struct{}
+	viewName    string
```

Comment:
No need for the redundant naming here.

```suggestion
	name    string
```